wshop-core.php

```diff
@@ -34,6 +79,11 @@ function wshop_underscore_templates() {
         // find all custom templates (defined in the current theme)
         $user_template_paths = glob(get_stylesheet_directory() . '/wshop-templates/*.php');
 
+        // exit if no paths found
+        if( count($user_template_paths <= 0) ){
```

Comment:
Probably just wrap the following for each loop in this if statement rather than returning